site/tutorials/gettings_started.md

```diff
@@ -0,0 +1,293 @@
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "bar",
+  "encoding": {
+    "y": {"field": "a", "type": "ordinal"},
+    "x": {"field": "b", "type": "quantitative", "aggregate": "mean"}
+  }
+}
+</div>
+
+## Customize your Visualization
+
+Vega-Lite automatically provides a large number of defaults, which you can override. For example, to set change the title of the x-axis from `MEAN(b)` to `average of b`, we can set the title property of the axis in the `x` channel.
+
+<div class="vl-example">
+{
+  "data": {
+    "values": [
+      {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+      {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+      {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+    ]
+  },
+  "mark": "bar",
+  "encoding": {
+    "y": {"field": "a","type": "ordinal"},
+    "x": {
+      "field": "b", "type": "quantitative", "aggregate": "mean",
+      "axis": {
+        "title": "average of b"
+      }
+    }
+  }
+}
+</div>
+
+
+## Publish your Visualization Online
+
+You can export visualizations as images from the [online editor](https://vega.github.io/vega-editor/?mode=vega-lite). However, if you want to embed your visualization on a website, you can create a web page with the following content.
+
+```html
+<!DOCTYPE html>
+<head>
+  <title>Vega Lite Bar Chart</title>
+  <meta charset="utf-8">
+
+  <script src="//d3js.org/d3.v3.min.js"></script>
+  <script src="//vega.github.io/vega/vega.js"></script>
+  <script src="//vega.github.io/vega-lite/vega-lite.js"></script>
+</head>
+<body>
+  <div id="vis"></div>
+  <script>
+  var vlSpec = {
+    "data": {
+      "values": [
+        {"a": "C", "b": 2}, {"a": "C", "b": 7}, {"a": "C", "b": 4},
+        {"a": "D", "b": 1}, {"a": "D", "b": 2}, {"a": "D", "b": 6},
+        {"a": "E", "b": 8}, {"a": "E", "b": 4}, {"a": "E", "b": 7}
+      ]
+    },
+    "mark": "bar",
+    "encoding": {
+      "y": {"field": "a","type": "ordinal"},
+      "x": {
+        "field": "b", "type": "quantitative", "aggregate": "mean",
+        "axis": {
+          "title": "average of b"
+        }
+      }
+    }
+  };
+
+  // Use Vega-Lite to compile the Vega-Lite specification to Vega
+  var vgSpec = vl.compile(vlSpec).spec;
+
+  // Use Vega to display the visualization
+  vg.parse.spec(vgSpec, function(chart) {
+    chart({el:"#vis"}).update();
+  });
+  </script>
+</body>
+</html>
+```
+
+As you can see, we created a variable `vlSpec` that holds the specification in JSON format.
+
+We also loaded the dependencies for Vega-Lite (D3, Vega and Vega-Lite) in the head of the document.
+
+In the JavaScript code, Vega-Lite compiles a Vega-Lite specification into a Vega specification. We can then use the [Vega Runtime](https://github.com/vega/vega/wiki/Runtime) to display visualization.
```

Comment:
`vl.compile` method compiles 


We can then use the Vega Runtime method `vg.parse`